examples/write_rundir.py

```diff
@@ -0,0 +1,18 @@
+import fv3config
```

Comment:
In the current master of fv3config (and in the version this repo uses in `requirements.txt`), you can run `write_run_directory <yaml_filename> <output_location>` to perform this command. If you use that, you can delete this file.

Reply:
write_run_directory keeps on failing for me.  Will stick with the existing Python script that I know works.  We can always go back to trying to use fv3config in another PR.

Reply:
We discussed how to fix this by updating the version of fv3config used to the latest master on fv3config's github repo. Can you please delete the file after that update?

Reply:
Removed and replaced in Jenkins plan with write_run_directory call from fv3config